app-admin/qtpass/qtpass-1.2.1.ebuild

```diff
@@ -25,25 +25,48 @@ RDEPEND="app-admin/pass
 	dev-qt/qtwidgets:5
 	net-misc/x11-ssh-askpass"
 DEPEND="${RDEPEND}
-	dev-qt/linguist-tools:5"
+	dev-qt/linguist-tools:5
+	dev-qt/qtsvg:5
```

Comment:
Please check if the package is installing `.svg` imagery, in that case it may be actually required in `RDEPEND` as well even if there is no linking.